.travis.yml

```diff
@@ -22,43 +11,19 @@ notifications:
 
 env:
   global:
-    - PYTHONWARNINGS=always::DeprecationWarning
```

Comment:
Are we losing the `PYTHONWARNINGS=always::DeprecationWarning` with this change or is it enabled elsewhere?

Reply:
This is only useful for running our own tests and is now specified in noxfile.py